src/pybind/mgr/ssh/__init__.py

```diff
@@ -1 +1,31 @@
-from .module import SSHOrchestrator
+from __future__ import absolute_import
+
+import os
+
+
+if 'UNITTEST' not in os.environ:
+    from .module import SSHOrchestrator
```

Comment:
Couldn't we patch the module where the tests are executed? Mixing the mocking code and the real code in one file doesn't look very nice imo.

Reply:
We have that problem that we can't import `.module` as some base classes are generated in C++ and thus will fails with a `NameError`. Do you have a better idea?

Reply:
hm. no.

Reply:
See #31561